src/config/locales/i18n.js

```diff
@@ -36,10 +37,10 @@ class I18n {
     }
 
     if (this.isArray(text)) {
-      return text.join('\n')
+      return removeMd(text.join('\n'))
     }
 
-    return text
+    return removeMd(text)
```

Comment:
This was added so that we could use translations in the error message testing like `i18n.t('error.date.min.title')`.

I ran into issues where the text was matching because of markdown being in the translation.
Adding the `removeMd` removes the `*` in this example `This is **some** text` which allows the text to match.

Reply:
Thanks – How will this change affect non-testing code?